pyxem/signals/electron_diffraction2d.py

```diff
@@ -175,7 +175,7 @@ def set_experimental_parameters(
                 exposure_time,
             )
 
-    def set_diffraction_calibration(self, calibration, center=None):
+    def set_diffraction_calibration(self, calibration, center=None, center_pixels=None):
```

Comment:
```suggestion
    def set_diffraction_calibration(self, calibration, center=None, pixel=False):
```

Reply:
Using a `bool` is a good suggestion, I looked into it earlier when you suggested it. I tried the other solution as I found that to be a little bit clearer in terms of argument names and implementation. If we want to go with a boolean argument, I think users might find `pixels` a bit counterintuitive (it can be interpreted in many ways). Of course the docstring explains the usage, but perhaps we can come up with a better argument name? What do you think? 

An alternative might be a type check as is used in Hyperspy? If two `float`s are passed as the center argument it is interpreted as given in calibrated units, while a pair of `int`s are interpreted as pixel units?

Reply:
> An alternative might be a type check as is used in Hyperspy?

With this I think about `signal.inav[...]`, etc, usage by the way.

Reply:
I agree, doing this is not completely straight forward. I like the idea of splitting it into two separate methods. The old `set_diffraction_calibration` could in principle be continued where it applies these two methods and the docstring could easily describe the order of operations etc. If you like, I could try and see if I can come up with a suggestion for it? However, I am a bit short on time and I will be away on paternity leave for a few months soon so I might go AWOL without much notice.

Reply:
> I was kind of hoping [hyperspy/hyperspy#2756](https://github.com/hyperspy/hyperspy/pull/2756) would be merged into hyperspy and that might fix this problem too :)

@CSSFrancis, yes, we should get this PR sorted along with https://github.com/hyperspy/hyperspy/pull/3385.
xref https://github.com/hyperspy/hyperspy/pull/2756#issuecomment-2169287069